coalib/output/Logging.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+import json
+import logging
+
+
+def configure_logging():
+    """
+    Configures the logging with hard coded dictionary.
+    """
+    import sys
+    import logging.config
+
+    logging.config.dictConfig({
+        'version': 1,
+        'handlers': {
+            'colored': {
+                'class': 'logging.StreamHandler',
+                'formatter': 'colored',
+                'stream': sys.stderr
+            }
+        },
+        'root': {
+            'level': 'DEBUG',
+            'handlers': ['colored']
+        },
+        'formatters': {
+            'colored': {
+                '()': 'colorlog.ColoredFormatter',
+                'format': '%(log_color)s[%(levelname)s]%(reset)s[%(asctime)s]'
+                          ' %(message)s',
+                'datefmt': '%X',
+                'log_colors': {
+                    'ERROR': 'red',
+                    'WARNING': 'yellow',
+                    'INFO': 'blue',
+                    'DEBUG': 'green'
+                }
+            }
+        }
+    })
+
+
+def configure_json_logging():
+    """
+    Configures logging as a JSON objects.
+    Returns a StringIO() that captures the logs as json.
+    """
+    import logging.config
+    import io
+
+    stream = io.StringIO()
+
+    logging.config.dictConfig({
+        'version': 1,
+        'handlers': {
+            'json': {
+                'class': 'logging.StreamHandler',
+                'formatter': 'json',
+                'stream': stream
+            }
+        },
+        'root': {
+            'level': 'DEBUG',
+            'handlers': ['json']
+        },
+        'formatters': {
+            'json': {
+                '()': 'coalib.output.Logging.JSONFormatters',
+            }
+        }
+    })
+    return stream
+
+
+class JSONFormatters(logging.Formatter):
+    """
+    JSON formatter for python logging
+    """
+
+    def format(record):
```

Comment:
you have to decorate it with `@staticmethod`, otherwise your function won't work and `record` is going to be filled with the `self` instance^^